modules/openy_gc_auth/modules/openy_gc_auth_yusa/src/Form/VirtualYUSALoginForm.php

```diff
@@ -247,7 +247,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       }
     }
     else {
-      if ($result['Status'] == 'Inactive') {
+      if (isset($result['Status']) && $result['Status'] == 'Inactive') {
```

Comment:
@hamrant @anpolimus I added a check here, as there was an issue with the server while I was testing this and it was throwing bad errors. This should resolve the issue if the YUSA provider is unreachable.